app/policies/ama_policy.rb

```diff
@@ -0,0 +1,11 @@
+class AMAPolicy < ApplicationPolicy
+  def update?
+    record.author == user
+  end
+
+  def destroy?
+    record.author == user || is_admin?
+  end
+
+  alias_method :create?, :update?
```

Comment:
@cinquemb @NuckChorris Uh... this is a good question. For now I think that yes, only an admin can create an AMA, but they'll be doing it as a registered user.

For example, if we're doing an AMA for George Clooney, an admin will create the AMA for George, including any introduction text for the post body that George provides. George will then be able to respond to questions in the AMA using his account without issue.

Does that make sense?